service/src/test/java/uk/nhs/adaptors/gp2gp/utils/CodeableConceptMapperMockUtil.java

```diff
@@ -0,0 +1,7 @@
+package uk.nhs.adaptors.gp2gp.utils;
+
+public class CodeableConceptMapperMockUtil {
+    public static final String NULL_FLAVOR_CODE = "<code nullFlavor=\"UNK\">\n"
+        + "    <originalText>4</originalText>\n"
+        + "</code>";
```

Comment:
Putting on a single line might help with the formatting headaches, at least for the mock ones.

```suggestion
    public static final String NULL_FLAVOR_CODE = "<code nullFlavor=\"UNK\"><originalText>4</originalText></code>";
```

Reply:
Optional improvement: 4 => "Mocked code" so it is more obvious where this value comes from